functions/src/index.ts

```diff
@@ -1,20 +1,21 @@
 import * as functions from 'firebase-functions';
```

Comment:
I'm 90% sure you can just changed this to
```suggestion
import functions from 'firebase-functions';
```

Reply:
I don't think you can do that if it's not a default export